openbb_platform/extensions/index/openbb_index/index_router.py

```diff
@@ -123,3 +123,14 @@ async def sp500_multiples(
 ) -> OBBject:
     """Historical S&P 500 multiples and Shiller PE ratios."""
     return await OBBject.from_query(Query(**locals()))
+
+
+@router.command(model="IndexSectors")
+async def sectors(
+    cc: CommandContext,
+    provider_choices: ProviderChoices,
+    standard_params: StandardParams,
+    extra_params: ExtraParams,
+) -> OBBject:
+    """Index Sectors. Sector weighting of an index."""
+    return await OBBject.from_query(Query(**locals()))
```

Comment:
The default example in the docstring results in an empty results,  so would be good to add the custom example here

Reply:
Yes, I will have to do a round of revisions now that we have that implemented.